bindings/rust/s2n-tls-sys/build.rs

```diff
@@ -187,8 +187,18 @@ fn build_vendored() {
 fn builder(libcrypto: &Libcrypto) -> cc::Build {
     let mut build = cc::Build::new();
 
+    if let Ok(cflags) = std::env::var("CFLAGS") {
+        // cc will read the CFLAGS env variable and prepend the compiler
+        // command with all flags and includes from it, which may conflict
+        // with the libcrypto includes we specify. To ensure the libcrypto
+        // includes show up first in the compiler command, we prepend our
+        // includes to CFLAGS.
+        std::env::set_var("CFLAGS", format!("-I {} {}", libcrypto.include, cflags));
```

Comment:
Is it okay to permanently modify CFLAGS with our libcrypto include?

Reply:
This is just for the duration of our own build script

Reply:
Is that documented somewhere? Rust starts a new process for each build?

Reply:
Well here's at least a stack overflow where someone tries to affect other builds on purpose and fails: https://stackoverflow.com/questions/57017066/how-do-i-set-environment-variables-with-cargo/66509742#66509742

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#life-cycle-of-a-build-script